lib/apipie_dsl/application.rb

```
@@ -198,9 +199,10 @@ def ignored?(klass, method = nil)
 
     def reload_documentation
       dsl_classes_paths.each do |file|
+        puts "loading #{file}"
         begin
           load file
-        rescue StandardError
+ #       rescue StandardError # todo enable in debug mode
```

this turned to be good when I was debugging apipie-dsl, it swallows errors of apipie-dsl itself

---

This can be ignored for now for using within a Rails application (Rails do it in a better way).